plugin/audit_log/audit_log.c

```diff
@@ -70,6 +70,7 @@ static char *audit_log_exclude_databases= NULL;
 static char *audit_log_include_databases= NULL;
 static char *audit_log_exclude_commands= NULL;
 static char *audit_log_include_commands= NULL;
+int64 audit_log_buffer_size_overflow = 0;
```

Comment:
@laurynas-biveinis  previous build was failing because `my_atomic_add64` expects the first parameter to be an int64 (not uint64 like it was before)